java-frontend/src/main/java/org/sonar/java/resolve/JavaType.java

```diff
@@ -492,8 +512,14 @@ public boolean isSubtypeOf(Type superType) {
     public boolean isSubtypeOfBound(JavaType type) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "isSubtypeOfBound" is 15 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)